src/cheri/riscv-priv-integration.adoc

```diff
@@ -447,7 +447,20 @@ NOTE: A future extension may add CHERI checks to the page table walker.
 
 === Modified Trap-Return Instructions Behavior
 
-When the {cheri_base_ext_name} base ISA is implemented, the <<otherpriv,trap-return instructions>> (`MRET` and `SRET`) read the full YLEN bits of the <<mepc_y>>/<<sepc_y>> register and unseal it prior to exception return if it is a <<sentry_cap>>.
+#Begin changed since last ARC review#
+
+When the {cheri_base_ext_name} base ISA is implemented
+and the platform incorporates {rvy_sentry_jalr_ext_name},
```

Comment:
Would just drop this condition. If it would be unsealed by JALR then we unseal in XRET as well.